app/models/institution_builder.rb

```diff
@@ -776,6 +777,88 @@ def self.build_versioned_school_certifying_official(version_id)
     SchoolCertifyingOfficial.connection.execute(sql)
   end
 
+  def self.build_institution_category_ratings_for_category(version_id, category)
+    sql = <<-SQL
+      INSERT INTO institution_category_ratings (
```

Comment:
Can this be broken up in any way? I don't have a great suggestion but it's definitely a large method

Reply:
I would be happy to, but I don't think I really can. It's one statement, and I can make it _shorter_ by making it harder to read but that seems like the wrong move for something like this...

Reply:
Agreed, sounds like a fair tradeoff